src/Avalonia.Controls/Primitives/Popup.cs

```diff
@@ -474,12 +474,12 @@ public void Open()
                         dismissLayer.IsVisible = false;
                         dismissLayer.InputPassThroughElement = null;
                     }).DisposeWith(handlerCleanup);
-                    
+
                     SubscribeToEventHandler<LightDismissOverlayLayer, EventHandler<PointerPressedEventArgs>>(
                         dismissLayer,
                         PointerPressedDismissOverlay,
-                        (x, handler) => x.PointerPressed += handler,
-                        (x, handler) => x.PointerPressed -= handler).DisposeWith(handlerCleanup);
+                        (x, handler) => x.AddHandler(PointerPressedEvent, handler, handledEventsToo:true),
```

Comment:
`handledEventsToo` shouldn't be used as a hack here

Reply:
I can revert it after decision about OnPointerPressed. Currently this change along with ListBoxItem is consequence of handling pointerpressed in base control. It needs to be handled because opening context menu should not trigger any other click actions on control, except two cases - selecting list items and closing popups.